adhocracy4/maps/widgets.py

```diff
@@ -22,9 +22,9 @@ def render(self, name, value, attrs):
 
         context = {
             'map_url': settings.BASE_MAP,
-            'bbox': json.dumps(settings.MAP_BOUNDING_BOX),
```

Comment:
I checked again in out code. And it seems that the setting value is stored as String. I would rather like to change this fact, to have this match the polygon.